### PR TITLE
Bug 1746103: Fix secret in pod log in 4.1

### PR DESCRIPTION
**Pull request by benjaminapetersen**
- Related to #276 
- Bump to library-go handles this per @spadgett comment on bugzilla, ref https://github.com/openshift/library-go/pull/472
- Pins `glide.yaml` to `release-4.1` for all `openshift` related repos.

### CODE REVIEW

**Comment by sfowl** [comment]
@benjaminapetersen Looks like this PR has not triggered a bugzilla update. Maybe a ':' suffix in the PR title and `/bugzilla retest` is needed?

**Comment by benjaminapetersen** [comment]
Bugzilla won't update https://bugzilla.redhat.com/show_bug.cgi?id=1746103 until the original bug for 4.2 https://bugzilla.redhat.com/show_bug.cgi?id=1737315 is validated.

One update made to operator code to fix a dependency caused reference change.  Tests should pass now.

File: glide.yaml [hunk 1]
```diff
@@ -17,13 +17,13 @@ import:
...
+  version: release-4.1
 - package: github.com/openshift/client-go
-  version: master
+  version: release-4.1
 - package: github.com/openshift/library-go
-  version: master
+  version: release-4.1
 
-# codegen
+  # codegen
```
Comment (spadgett) [inline glide.yaml]: nit: unintentional whitespace change?

**Pull request merged**